packages/commonwealth/client/scripts/views/pages/create_community/chain_input_rows.tsx

```diff
@@ -135,6 +153,8 @@ export const ethChainRows = (
     app?.user.isSiteAdmin ? { label: 'Custom', value: 'Custom' } : {},
   ] as Array<DropdownItemType>;
 
+  const ethChainNode = options.filter((o) => o.label === 'Ethereum Mainnet')[0];
```

Comment:
What does this change do?

Reply:
Can we use some other criterion than the plaintext label, in case we want to rename the node? @kurtisassad